lib/libzrepl/data_conn.c

```diff
@@ -242,7 +242,8 @@ uzfs_submit_writes(zvol_info_t *zinfo, zvol_io_cmd_t *zio_cmd)
 
 #ifdef DEBUG
 	if (is_rebuild) {
-		ASSERT(ZVOL_IS_REBUILDING(zinfo->main_zv));
+		ASSERT(ZVOL_IS_REBUILDING(zinfo->main_zv) ||
+		    (zinfo->main_zv->rebuild_info.rebuild_cnt > 1));
```

Comment:
why this new check?

Reply:
instead of checking for rebuild_cnt, we can probably have 'or' check for rebuild status being errored.. however, this also should be fine to see that rebuild is in progress

Reply:
ok.